documentation/content/en/books/arch-handbook/boot/_index.adoc

```diff
@@ -57,7 +57,7 @@ The FreeBSD boot process can be surprisingly complex. After control is passed fr
 [[boot-overview]]
 == Overview
 
-The boot process is an extremely machine-dependent activity. Not only must code be written for every computer architecture, but there may also be multiple types of booting on the same architecture. For example, a directory listing of [.filename]#/usr/src/sys/boot# reveals a great amount of architecture-dependent code. There is a directory for each of the various supported architectures. In the x86-specific [.filename]#i386# directory, there are subdirectories for different boot standards like [.filename]#mbr# (Master Boot Record), [.filename]#gpt# (GUID Partition Table), and [.filename]#efi# (Extensible Firmware Interface). Each boot standard has its own conventions and data structures. The example that follows shows booting an x86 computer from an MBR hard drive with the FreeBSD [.filename]#boot0# multi-boot loader stored in the very first sector. That boot code starts the FreeBSD three-stage boot process.
+The boot process is an extremely machine-dependent activity. Not only must code be written for every computer architecture, but there may also be multiple types of booting on the same architecture. For example, a directory listing of [.filename]#/usr/src/stand# reveals a great amount of architecture-dependent code. There is a directory for each of the various supported architectures. In the x86-specific [.filename]#i386# directory, there are subdirectories for different boot standards like [.filename]#mbr# (Master Boot Record), [.filename]#gptboot# (GUID Partition Table), and [.filename]#zfs# ((previously: Zettabyte file system) Combination of a file system with a volume manager). The [.filename]#efi# (Expandable Firmware Interface) has stayed out of this folder in general [.filename]#stand/#. Each boot standard has its own conventions and data structures. The example that follows shows booting an x86 computer from an MBR hard drive with the FreeBSD [.filename]#boot0# multi-boot loader stored in the very first sector. That boot code starts the FreeBSD three-stage boot process.
```

Comment:
There's a full path used in this this.

Reply:
I have left it as it is in the original document full path used in this one.It Isn't hard for me to fix it if I have to?

Reply:
"MBR" isn't the standard. It's called "CSM boot" these days. It happens to use MBR to implement it, though.
GPT also is a partitioning standard, not a booting standard. And ZFS isn't a boot standard, but a filesystem.
EFI is more covnetionally referred to UEFI since the EFI standard is 20 years obsolete.

So we support CSM (with both GPT and MBR partitioning support) and UEFI booting (GPT is totally supported, MBR is mostly supported). We support loading files from ext2fs, MSDOS, UFS and ZFS. We also support the boot environment feature of ZFS which allows the HOST OS (FreeBSD) to communicate details about what to boot that go beyond a simple partition which was possible in the past. 

I get that it may be hard to convey these nuances, but the above paragraph needs additional work since it's written with the older sensibilities and reads as somewhat dated.

Finally, the docs are moving to 'One sentence per line' rather than 'One paragraph per line' style...

Reply:
So this is a suggestion on how to implement your proposal: 
![Uploading boot.png…]()
About 'One sentence per line' rather than 'One paragraph per line' style, I would leave the @debdrup to tidy up as promised?